core/unit_test/TestDefaultDeviceTypeInit.hpp

```diff
@@ -431,6 +430,7 @@ TEST(defaultdevicetypeinit, commandline_args_nthreads_numa_device_other) {
   delete[] args;
 }
 #endif
+#endif
```

Comment:
???

Reply:
Did you mean to commit this?

Reply:
Looks like an extra #endif to me

Reply:
Leftover from reenabling the default tests. Fixed and rebased.